site/src/components/table.md

```diff
@@ -86,17 +86,28 @@ For cases in which a table has many columns, the `table-wrapper` class is availa
 				<td>Cell</td>
 				<td>Cell</td>
 			</tr>
+			<tr>
+				<td>Row three</td>
+				<td>Cell</td>
+				<td>Cell</td>
+				<td>Cell</td>
+				<td>Cell</td>
+				<td>Cell</td>
+				<td>Cell</td>
+			</tr>
 		</tbody>
 	</table>
 </div>
-```
 
 ## Modifiers
 
-_`table-stacked-mobile`_ - stacks table cells on mobile screen sizes.
+All modifier classes should be applied in conjunction with the `.table` class.
 
-```html
-<table class="table table-stacked-mobile">
+### Simple
+
+_`table-simple`_ gives table has only horizontal cell borders.
```

Comment:
I think we may want to tweak the writing a bit here:

```suggestion
_`table-simple`_ gives the table only horizontal cell borders.
```

Reply:
🤦🏻‍♀️

Reply:
Updated